pkg/cmd/install.go

```diff
@@ -230,3 +235,43 @@ func (o *installCmdOptions) waitForPlatformReady(platform *v1alpha1.IntegrationP
 
 	return watch.HandlePlatformStateChanges(o.Context, platform, handler)
 }
+
+func (o *installCmdOptions) validate(cmd *cobra.Command, args []string) error {
+	var result error
+
+	// Let's register only our own APIs
+	schema := runtime.NewScheme()
+	if err := apis.AddToScheme(schema); err != nil {
+		return err
+	}
+
+	if o.contexts == nil {
+		return nil
+	}
+	for _, context := range o.contexts {
+		err := errorIfContextIsNotAvailable(schema, context)
```

Comment:
This does not take into account the "none" context

Reply:
Valid point. I added a check for "none", too now so that is a valid input.